### ISSUE TITLE
Make Stargate image configurable

### ISSUE THREAD

**Issue by jsanda** [k8ssandra/k8ssandra-operator#25]
This an extension of #23 but focuses on the Stargate CRD. I am breaking this out into a separate ticket because the work can happen separately and possibly in parallel.

┆Issue is synchronized with this [Jiraserver Task](https://k8ssandra.atlassian.net/browse/K8SSAND-632) by [Unito](https://www.unito.io)
┆Epic: Stargate Controller Integration
┆Fix Versions: k8ssandra-2.0.0-alpha
┆Issue Number: K8SSAND-632
┆Priority: Medium
